MobileFilter/sections/specific_web.txt

```diff
@@ -28,6 +28,8 @@ theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,
 cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,siirtpress.com,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
 !
 news24.jp##.adText > ul
+game.watch.impress.co.jp##aside[class^="ad"]
```

Comment:
This rule has false positive on desktop site (viewable on mobile too). Please change to `watch.impress.co.jp##.ad.movie`.